### PR TITLE
chore(ci): Fix nightly and verify jobs

### PR DESCRIPTION
**Pull request by jamtur01**

### CODE REVIEW

**Comment by jszwedko** [comment]
@jamtur01 looks like the Windows one still isn't happy:

https://github.com/timberio/vector/runs/1651418245?check_suite_focus=true

```
Run 'C:\Program Files\Vector\bin\vector --version'
The filename, directory name, or volume label syntax is incorrect.
Error: Process completed with exit code 1.
```

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -304,7 +301,7 @@ jobs:
       - name: "Run Vector"
         shell: cmd
         run: |
-          C:\Program Files\Vector\bin\vector --version
+          'C:\Program Files\Vector\bin\vector --version'
```
Comment (jszwedko) [inline .github/workflows/nightly.yml:304]: I think you just want quotes around `C:\Program Files\Vector\bin\vector`?

**Comment by jszwedko** [comment]
Seems like that wasn't it either :slightly_frowning_face: 

```
Run 'C:\Program Files\Vector\bin\vector' --version
The filename, directory name, or volume label syntax is incorrect.
Error: Process completed with exit code 1.
```

**Comment by jamtur01** [comment]
Yeah - I wish this was CircleCI and I could SSH into the damn host and see for myself.

**Pull request merged**